packages/manager/apps/web/client/app/components/manager-order-contracts/manager-order-contracts.controller.js

```diff
@@ -0,0 +1,11 @@
+export default class {
+  /* @ngInject */
+  constructor($element, $timeout) {
+    this.$element = $element;
+    this.$timeout = $timeout;
+  }
+
+  $postLink() {
+    this.$timeout(() => this.$element.removeAttr('name'));
```

Comment:
We have https://github.com/ovh-ux/ng-ovh-contracts for this purpose, why recreate a component ?

Reply:
Hmm not the same purpose, but maybe the component name is not valid, since this one's purpose is to display user conditions and agreements

Reply:
The purpose is to display a checkbox for a list of contracts ? 
If it's a generic component it should be share globally in `ng-ovh-contracts`.

Reply:
Related : https://github.com/ovh-ux/ng-ovh-contracts/pull/36

Reply:
I misunderstood the point, sorry, thanks @marie-j